app/models/tasks/fix_media_task.rb

```diff
@@ -0,0 +1,33 @@
+class Tasks::FixMediaTask < ::Task
+  before_save :update_media_resource, if: ->(t) { t.status_changed? && t.media_resource }
+
+  def media_resource
+    owner
+  end
+
+  def source_url
+    media_resource&.href
+  end
+
+  def porter_tasks
+    [
+      {
+        Type: "Transcode",
+        Format: "INHERIT",
+        Destination: {
+          Mode: "AWS/S3",
+          BucketName: ENV["FEEDER_STORAGE_BUCKET"],
+          ObjectKey: porter_escape(media_resource.path)
```

Comment:
Note this ends up overwriting our copy of the original in S3.  But I think this is fine?  I can't think of a reason we'd ever want/need it.  And we can always go back to the source url.

It'll also be easy to monitor these bad mp3s, by querying `Tasks::FixMediaTask.count`.

Reply:
I think it is fine? it's such a minimal change. it preserves id3 even, right?